src/Pages/Shared/_Layout.cshtml

```diff
@@ -11,117 +11,121 @@
 
-            }
-            else
-            {
-                <li class="nav-item"><a asp-page="/Login" class="btn btn-primary">Sign In</a></li>
-            }
+                }
+                else
+                {
+                    <li class="nav-item"><a asp-page="/Login" class="btn btn-primary">Sign In</a></li>
+                }
 
-        </ul>
-    </div>
-</nav>
-<div class="content">
-    @RenderBody()
-    @await RenderSectionAsync("Scripts", false)
-</div>
-<div class="container-fluid footer">
-        <a href="https://twitter.com/vu_lt" class="footer-image fab fa-twitter"></a>
-        <a href="https://www.facebook.com/VilniusUniversity/" class="footer-image fab fa-facebook-f"></a>
-        <a href="https://www.instagram.com/vilniusuniversity/" class="footer-image fab fa-instagram"></a>
+                </ul>
+            </div>
+        </nav>
+        <div class="content">
+            @RenderBody()
+            @await RenderSectionAsync("Scripts", false)
+        </div>
+        <div class="container-fluid footer">
+            <a href="https://twitter.com/vu_lt" class="footer-image fab fa-twitter"></a>
+            <a href="https://www.facebook.com/VilniusUniversity/" class="footer-image fab fa-facebook-f"></a>
+            <a href="https://www.instagram.com/vilniusuniversity/" class="footer-image fab fa-instagram"></a>
 
-        <p>© Vilnius University</p>
-    </div>
-<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.min.js" integrity="sha384-w1Q4orYjBQndcko6MimVbzY0tgp4pWB4lZ7lr30WKz0vr/aWKhXdBNmNb5D92v7s" crossorigin="anonymous"></script>
-    <!-- Font Awesome  -->
-    <script src="https://kit.fontawesome.com/f58a6230c2.js" crossorigin="anonymous"></script>
+            <p>© Vilnius University</p>
+        </div>
+        <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.min.js" integrity="sha384-w1Q4orYjBQndcko6MimVbzY0tgp4pWB4lZ7lr30WKz0vr/aWKhXdBNmNb5D92v7s" crossorigin="anonymous"></script>
+        <!-- Font Awesome  -->
+        <script src="https://kit.fontawesome.com/f58a6230c2.js" crossorigin="anonymous"></script>
 
-    <script>
+        <script>
     var snow = @Context.Request.Cookies.TryGetValue("Snow", out var value).ToString().ToLower();
     if (snow) {
         document.addEventListener("DOMContentLoaded",
             function() {
                 particlesJS.load('particles-js', 'js/particle-cfg.json');
             });
     };
-    </script>
+        </script>
+
+
+        <script src="~/js/site.js"></script>
```

Comment:
Very minor detail, but I think 2 empty lines are unnecessary, 1 would be enough :D

Reply:
Agree, did this by mistake, so in new update I also removed some old commented code while I had the chance :D